packages/react-file-type-icons/src/initializeFileTypeIcons.tsx

```diff
@@ -6,7 +6,7 @@ import type { IIconOptions } from '@fluentui/style-utilities';
 const PNG_SUFFIX = '_png';
 const SVG_SUFFIX = '_svg';
 
-export const DEFAULT_BASE_URL = 'https://spoppe-b.azureedge.net/files/fabric-cdn-prod_20221015.001/assets/item-types/';
+export const DEFAULT_BASE_URL = 'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230106.001/assets/item-types/';
```

Comment:
FYI, we will want to switch to `res.cdn.office.net` at some point as that is the "actual" primary endpoint as of now. But that can be done separately as part of a broader push.

Reply:
Makes sense. I need to learn the diff between res and res-1. Based on your comment i'll mark this as resolved and plan for the next PR or a broader push.